tatanka/client/conn/conn.go

```diff
@@ -421,40 +522,39 @@ func (c *MeshConn) tatankas(mode TatankaSelectionMode) (tts []*tatanka, _ error)
 	}
 }
 
-func (c *MeshConn) handleTatankaMessage(tatankaID tanka.PeerID, msg *msgjson.Message) *msgjson.Error {
+func (c *MeshConn) handleTatankaMessage(tatankaID tanka.PeerID, msg *msgjson.Message) {
```

Comment:
Why are getting rid of these error returns?

Reply:
The return values were being ignored. I think if we need it, we can add it back, but generally handlers don't have return values. `http.HandlerFunc` doesn't have a return value either.